### DISCUSSION TITLE
Provide context phrases/words to NeMo ASR

### DISCUSSION THREAD

**Discussion by Herajy** [NVIDIA-NeMo/NeMo#3091] [Q&A]
Hi,

Is it possible to provide NeMo ASR with context phrases that can help  increasing the accuracy of resulting text.

In other words, suppose I know in advance that the audio file which needs to be transcripted contains certain words/phrases. Can NeMo ASR take advantage of that?

**Comment by titu1994** [comment]
We don't support that is in Nemo yet, but we are looking into personalized ASR

**Comment by Herajy** [comment]
many thanks for answering this
